buildSrc/src/main/java/Config.kt

```diff
@@ -1,10 +1,10 @@
 import java.math.BigDecimal
 
 object Config {
-    val kotlinVersion = "1.4.10"
+    val kotlinVersion = "1.4.30"
```

Comment:
if I don't bump it, I fall into https://github.com/Kotlin/dokka/issues/1272

Modules that use Kotlin 1.2 or 1.3 still fallback to the desired version, see `springKotlinCompatibleLanguageVersion` flag